tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/Transports/LargePayload/DefaultTransportLargePayloadTests.cs

```diff
@@ -8,6 +8,7 @@
 
 namespace Datadog.Trace.ClrProfiler.IntegrationTests
 {
+    [Collection("LargePayloadTests")]
```

Comment:
tiniest nit: my aversion to magic strings would lead me to do this instead 😄 
```suggestion
    [Collection(nameof(LargePayloadTestBase))]
```